proposals/kubeflow-steering-committee-charter.md

```diff
@@ -0,0 +1,118 @@
+# Kubeflow Steering Committee
+The Kubeflow Steering Committee (KSC) is the governing body of the Kubeflow project, providing decision-making and oversight pertaining to the Kubeflow project policies, sub-organizations, and financial planning, and defines the project values and structure.
+
+The governance of Kubeflow is an open, living document, and will continue to evolve as the community and project change.
+
+### Charter
+
+1. Define, evolve, and promote the vision, values, and mission of the Kubeflow project.
+1. Define and evolve project and group governance structures and policies, including election rules, working group policies, and project roles.
+1. Steward, control access, delegate access, and establish processes regarding all Kubeflow project resources and have the final say in the disposition of those resources.
+1. Define and evolve the scope of the Kubeflow community, including acceptance of new projects into Kubeflow.
```

Comment:
@james-jwu I suspect that the KSC will need to manage a budget and to make financial and legal commitments on behalf of the project.    I think we may need to address this potential requirement.

Reply:
@andreyvelich Yes to "KSC has the power to define the scope of the individual Working Groups". This responsibility is can be delegated to the TOC. For project acceptance, I think TOC and KSC should align, but KSC will have the power to approve. For WG roadmap I think this will be delegated to TOC.

Reply:
@jbottum Based on information from @theadactyl, CNCF will be managing budget, although I am not familiar with the details here. For legal issues, CNCF will provide support: https://www.cncf.io/services-for-projects/#legal-services

Maybe here we can add "Collaborate with CNCF on financial planning and legal issues that involves Kubeflow".

Reply:
>Yes to "KSC has the power to define the scope of the individual Working Groups". 

@james-jwu In that case, what would be the differences between WG Co-chairs and KSC in terms of establishing scope ?
How the Roadmap discussions should be addressed and be selected ?